docs/using-ci-builds.md

```diff
@@ -0,0 +1,41 @@
+# Using CI Builds
+
+Master branch build artifacts are stored [here](https://ci.appveyor.com/project/dxrobot/devextreme-aspnet-data/branch/master/artifacts). Download the `devextreme-aspnet-data-99.0.0-ci-NNN.tgz` file and use the instruction for the client-side script, or get `DevExtreme.AspNet.Data.99.0.0-ci-NNN.nupkg` and follow the one for the NuGet package.
+
+## Client-Side Script
+
+Unpack the `.tgz` archive to a temporary location. On Windows, you can use [7zip](http://www.7-zip.org/download.html) for that:
+
+```bash
+7zip x devextreme-aspnet-data-99.0.0-ci-NNN.tgz
+7zip x devextreme-aspnet-data-99.0.0-ci-NNN.tar
+```
+
+Navigate to the package/js folder in this location and copy the extracted JavaScript file to your project.
```

Comment:
"this location" is not clear for me.
Can we rephrase in terms of "extracted files" or something like this?

Reply:
Well, it states "Unpack the archive to a **temporary location**" a couple of lines before, what can be unclear?

Reply:
Maybe it's just me.